tools/cassandra/README.md

```diff
@@ -6,19 +6,26 @@ How
 ---
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
+- Make sure you already created cadence and visibility keyspaces (see `keyspace.cql` in schema)
 
 Setting up initial cassandra schema on a new cluster
 ----------------------------------------------------
 ```
 ./cadence-cassandra-tool -ep 127.0.0.1 -k cadence setup-schema -v 0.0 -- this sets up just the schema version tables with initial version of 0.0
```

Comment:
There is a make target for this. You can just replace these with
make install-schema

Reply:
make sense, added